src/pages/DistributionsPage/DistributionsPage.tsx

```diff
@@ -100,21 +101,35 @@ export function DistributionsPage() {
       d.distribution_type === DISTRIBUTION_TYPE.COMBINED
   );
 
-  const isCombinedDistribution =
-    fixedDist &&
-    circleDist &&
-    circleDist.distribution_type === DISTRIBUTION_TYPE.COMBINED &&
-    fixedDist.distribution_type === DISTRIBUTION_TYPE.COMBINED;
-
+  const getUserClaimedFixedPaymentAmt = (
+    dist: typeof epoch.distributions[0],
+    address?: string
+  ) => {
+    if (!address) return 0;
+    return dist.distribution_json?.fixedGifts &&
+      address in dist.distribution_json.fixedGifts
+      ? Number(
+          formatUnits(
+            dist.distribution_json.fixedGifts[address],
+            dist.vault.decimals
+          )
+        )
+      : 0;
+  };
   const unwrappedAmount = (
     id?: number,
     dist?: typeof epoch.distributions[0]
   ) => {
-    if (!id || !dist) return 0;
-    return (
-      (dist.claims.find(c => c.profile?.id === id)?.new_amount || 0) *
-      dist.pricePerShare.toUnsafeFloat()
-    );
+    if (!id || !dist) return { claimed: 0, fixedPayment: 0, circleClaimed: 0 };
+    const claim = dist.claims.find(c => c.profile?.id === id);
+    const pricePerShare = dist.pricePerShare.toUnsafeFloat();
+    const fixedPaymentAmt = getUserClaimedFixedPaymentAmt(dist, claim?.address);
+    return {
+      claimed: (claim?.new_amount || 0) * pricePerShare,
+      fixedPayment: fixedPaymentAmt * pricePerShare,
+      circleClaimed:
+        ((claim?.new_amount || 0) - fixedPaymentAmt) * pricePerShare,
```

Comment:
Same goes for here: Why not do BigNumber math here?

Reply:
tried using `BigNumber` but `pricePerShare` is always has decimals and would cause an underflow 
or should i just change it to using a `FixedNumber` instead

Reply:
the amount & new_amount columns are unfortunately already lossy due to bad support for the NUMERIC postgres column type in Zeus. this is one of those pieces of vault tech debt i was talking about. so there's nothing to be gained in being more precise with calculations here until we fix that issue. (when we have to be exact, i.e. interactions with the smart contracts, we use the values stored in `distributions.distribution_json` instead)